src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Nls.cs

```diff
@@ -287,12 +287,12 @@ private static int[] ConvertWin32GroupString(string win32Str)
             // None of these cases make any sense
             if (string.IsNullOrEmpty(win32Str))
             {
-                return new int[] { 3 };
+                return [3];
```

Comment:
Just a note for any other reviewers that these get cached in a static readonly by a higher level caller.